certbot/cli.py

```diff
@@ -956,6 +956,11 @@ def prepare_and_parse_args(plugins, args, detect_defaults=False):  # pylint: dis
         help="Show tracebacks in case of errors, and allow certbot-auto "
              "execution on experimental platforms")
     helpful.add(
+        ["certonly", "renew", "run"], "--debug-challenges", action="store_true",
```

Comment:
I'd add `None` as the first element in this list. This changes where `--debug-challenges` is displayed in the output of `--help all`. Currently, it's listed under `certonly`, but it's more of a general flag for a lot of what Certbot does, so I'd use `None` which puts it under "Optional arguments".